code/modules/mob/living/carbon/human/life/handle_mutations_and_radiation.dm

```diff
@@ -73,3 +89,200 @@
+				var/inst = pick(1, 2, 3)
+				switch(inst)
+
+					if(1)
+						//Drop some meat
+						to_chat(src, "<span class='warning'>A chunk of meat falls off of you!</span>")
+						var/totalslabs = 1
+						var/obj/item/weapon/reagent_containers/food/snacks/meat/allmeat[totalslabs]
+						var/sourcename = real_name
+						var/sourcejob = job
+						var/sourcenutriment = nutrition / 15
+						//var/sourcetotalreagents = mob.reagents.total_volume
+
+						for(var/i = 1 to totalslabs)
+							var/obj/item/weapon/reagent_containers/food/snacks/meat/human/newmeat = new
+							newmeat.name = sourcename + " " + newmeat.name
+							newmeat.subjectname = sourcename
+							newmeat.subjectjob = sourcejob
+							newmeat.reagents.add_reagent(NUTRIMENT, sourcenutriment / totalslabs) //Thehehe. Fat guys go first
+							//src.occupant.reagents.trans_to(newmeat, round (sourcetotalreagents / totalslabs, 1)) // Transfer all the reagents from the
+							allmeat[i] = newmeat
+
+							var/obj/item/meatslab = allmeat[i]
+							var/turf/Tx = get_turf(src)
+							meatslab.forceMove(get_turf(src))
+							meatslab.throw_at(Tx, i, 3)
+
+							if(!Tx.density)
+								var/obj/effect/decal/cleanable/blood/gibs/D = getFromPool(/obj/effect/decal/cleanable/blood/gibs, Tx)
+								D.New(Tx,i)
+
+					if(2)
+						//Drop a limb
+						var/list/datum/organ/external/candidates = list()
+						for (var/datum/organ/external/O in organs)
+							if(O.status & ORGAN_ROBOT)
+								continue
+							if(O.vital)
+								continue
+							if(O.amputated)
+								continue
+							candidates.Add(O)
+						if (candidates.len)
+							var/datum/organ/external/victim = pick(candidates)
+							victim.droplimb(1)
+					if(3)
+						//Skeletonify a limb
+						var/list/datum/organ/external/candidates = list()
+						for (var/datum/organ/external/O in organs)
+							if(O.status & ORGAN_ROBOT)
+								continue
+							if(O.amputated)
+								continue
+							if(O.species == "Skeletal Vox" || O.species == "Skellington")
+								continue
+							candidates.Add(O)
+						if(candidates.len)
+							var/datum/organ/external/victim = pick(candidates)
+							to_chat(src, "<span class = 'warning'>The flesh is falling off of your [victim.display_name]!</span>")
+							var/new_species_name
+							if(isvox(src))
+								new_species_name = "Skeletal Vox"
+							else
+								new_species_name = "Skellington"
+							var/datum/species/S = all_species[new_species_name]
+							victim.species = new S.type
+							update_body()
+			if(prob(1*major_rad_multiplier))
+				//Rad glow
+				if(!(species.flags & RAD_GLOW))
+					to_chat(src, "<span class = 'blob'>You start glowing!</span>")
+					species.flags |= RAD_GLOW
+		if(rad_tick > 600)
+			if(prob(0.01*extreme_rad_multiplier))
+				//Ghoulification
+				to_chat(src, "<span class = 'notice'>You feel strangely at peace.</span>")
```

Comment:
Should probably put this message after the if, otherwise it doesn't really mean anything.